### PR TITLE
release notes

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

File: app/views/pages/release-notes/2020/JANUARY/13.liquid [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+converter: markdown
+metadata:
+  title: error handling and GraphQL fixes
```
Comment (pavelloz) [inline app/views/pages/release-notes/2020/JANUARY/13.liquid]:
```suggestion
  title: Error handling and GraphQL fixes
```

File: app/views/pages/release-notes/release-notes.liquid [hunk 2]
```diff
@@ -17,6 +17,22 @@ searchable: false
...
 
+<div class="row release-notes__year-container">
+  <div class="col-auto release-notes__timeline">
+    <h3>2020</h3>
+  </div>
+
+  <div class="col">
+    <ul class="list-unstyled">
+      <li>
+        <a href="/release-notes/13-JAN-2020">error handling and GraphQL fixes</a>
```
Comment (pavelloz) [inline app/views/pages/release-notes/release-notes.liquid]:
```suggestion
        <a href="/release-notes/13-JAN-2020">Error handling and GraphQL fixes</a>
```

File: app/views/pages/release-notes/2020/JANUARY/13.liquid [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  skip_contribute_button: true
+slug: release-notes/13-JANUARY-2020
+searchable: true
+---
+
+#### {{ page.metadata.description }}
+
+<h4 class="release-note release-note__improved">IMPROVED</h4>
+
+* show a warning in pos-cli logs when adding/updating a profile via GraphQL/form  which do not exist
```
Comment (pavelloz) [inline app/views/pages/release-notes/2020/JANUARY/13.liquid]:
```suggestion
* show a warning in `pos-cli logs` when adding/updating a profile via GraphQL/form which do not exist
```

File: app/views/pages/release-notes/2020/JANUARY/13.liquid [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+* error handling of BadRequest, for example when invalid JSON is provided as parameters to Page
+* data export will include user level properties as well
+
+<h4 class="release-note release-note__fixed">FIXED</h4>
+
+* assigning properties to user profiles in some scenarios
+* sorting by updated_at in some admin_* queries
```
Comment (pavelloz) [inline app/views/pages/release-notes/2020/JANUARY/13.liquid]:
```suggestion
* sorting by `updated_at` in some admin_* queries
```

File: app/views/pages/release-notes/2020/JANUARY/13.liquid [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+* jwt_decode_and_set_session tag will correctly set context.current_user
```
Comment (pavelloz) [inline app/views/pages/release-notes/2020/JANUARY/13.liquid]:
```suggestion
* `jwt_decode_and_set_session` tag will correctly set `context.current_user`
```

File: app/views/pages/release-notes/2020/JANUARY/13.liquid [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+* show a warning in `pos-cli logs` when adding/updating a profile via GraphQL/form which do not exist
+* error handling of BadRequest, for example when invalid JSON is provided as parameters to Page
+* data export will include user level properties as well
+
+<h4 class="release-note release-note__fixed">FIXED</h4>
+
+* assigning properties to user profiles in some scenarios
+* sorting by `updated_at` in some admin_* queries
+* `jwt_decode_and_set_session` tag will correctly set `context.current_user`
```
Comment (pavelloz) [inline app/views/pages/release-notes/2020/JANUARY/13.liquid]: Tag? Isnt it a mutation or smth?

Comment (diana-lakatos) [reply app/views/pages/release-notes/2020/JANUARY/13.liquid]: @Slashek

Comment (Slashek) [reply app/views/pages/release-notes/2020/JANUARY/13.liquid]: oh you are correct, will improve

Comment (Slashek) [inline app/views/pages/release-notes/2020/JANUARY/13.liquid]:
```suggestion
* [jwt_decode_and_set_session](/api-reference/graphql/mutations#jwt_decode_and_set_session) mutation will correctly set `context.current_user`
```

**Comment by diana-lakatos** [comment]
Reviewed and edited, thank you!

**Pull request merged**